readme.md

```diff
@@ -27,7 +27,23 @@ console.log(wrapAnsi(input, 20));
 
 ## API
 
-### wrapAnsi(input, columns)
+### wrapAnsi(input, columns, opts)
+
+wrap words to the specified column width. By default the wrap is
+`soft`, so long words may extend past the column length.
+
+the following options can be provided:
+
+* `hard`: `boolean`, should hard wrapping be enabled?
+
+### wrapAnsi.hard(input, columns)
+
+long words will be broken up so that they do not extend past the column width.
+Equivalent to `{hard: true}`.
+
+### wrapAnsi.soft(input, columns)
```

Comment:
I think I've addressed most of the other review ... how are folks feeling, should we drop the `.hard` and `.soft`? this was modeled after substack's wordwrap API, I don't feel too strongly though -- happy to simply have an opts object.